ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py

```diff
@@ -1468,3 +1468,37 @@ def test_paddle_max(
         axis=axis,
         keepdim=False,
     )
+
+
+# heaviside
+@handle_frontend_test(
+    fn_tree="paddle.tensor.math.heaviside",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```

Comment:
If it can take integers as well, this should be `numeric` so that all acceptable dtypes are tested instead of floats only